resources/build/increment-version.sh

```diff
@@ -192,7 +192,7 @@ if [ "$action" == "commit" ]; then
       git commit -m "$message (history cherry-pick to master)"
       # TODO: once we are sure this is stable, add `-l auto` to get the "auto:"
       # label
-      hub pull-request -f --no-edit -b master
+      hub pull-request -fp --no-edit
```

Comment:
So this doesn't use -b stable-17.0

Reply:
correct -- this is supposed to write the same history changes to HISTORY.md on master (thus avoiding the need for manual updates of HISTORY.md on master in the future...)